src/parser/parse_op.h

```diff
@@ -69,12 +70,16 @@ struct ParseOp
   api::Sort d_type;
   /** The operator associated with the parsed operator, if it exists */
   api::Op d_op;
+  /**
+   * The indices if the operator is indexed, but api::Op is the null operator
```

Comment:
Can go a bit more into detail here? We only need this for operators that can be applied to different sorts with different arguments. So far, this would only be `to_fp`.